spinaltap-mysql/src/main/java/com/airbnb/spinaltap/mysql/binlog_connector/BinaryLogConnectorEventMapper.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright 2018 Airbnb. Licensed under Apache-2.0. See License in the project root for license
+ * information.
+ */
+package com.airbnb.spinaltap.mysql.binlog_connector;
+
+import com.airbnb.spinaltap.mysql.BinlogFilePos;
+import com.airbnb.spinaltap.mysql.event.BinlogEvent;
+import com.airbnb.spinaltap.mysql.event.DeleteEvent;
+import com.airbnb.spinaltap.mysql.event.QueryEvent;
+import com.airbnb.spinaltap.mysql.event.StartEvent;
+import com.airbnb.spinaltap.mysql.event.TableMapEvent;
+import com.airbnb.spinaltap.mysql.event.UpdateEvent;
+import com.airbnb.spinaltap.mysql.event.WriteEvent;
+import com.airbnb.spinaltap.mysql.event.XidEvent;
+import com.github.shyiko.mysql.binlog.event.DeleteRowsEventData;
+import com.github.shyiko.mysql.binlog.event.Event;
+import com.github.shyiko.mysql.binlog.event.EventHeaderV4;
+import com.github.shyiko.mysql.binlog.event.EventType;
+import com.github.shyiko.mysql.binlog.event.QueryEventData;
+import com.github.shyiko.mysql.binlog.event.TableMapEventData;
+import com.github.shyiko.mysql.binlog.event.UpdateRowsEventData;
+import com.github.shyiko.mysql.binlog.event.WriteRowsEventData;
+import com.github.shyiko.mysql.binlog.event.XidEventData;
+import java.util.Optional;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+/**
+ * Represents a mapper that maps a {@link com.github.shyiko.mysql.binlog.event.Event} to a {@link
+ * com.airbnb.spinaltap.mysql.event.BinlogEvent}.
+ */
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public final class BinaryLogConnectorEventMapper {
```

Comment:
Was previously in `MysqlSource` (now `BinaryLogConnectorSource`). Separated the mapping logic for clarity and reusability.